tasks/changelog.py

```diff
@@ -22,6 +22,8 @@
 # match something like `(#1234)` and return `1234` in a group
 PR_REG = re.compile(r'\(\#(\d+)\)')
 
+NO_CHANGELOG_LABEL = 'no-changelog'
```

Comment:
let's call it `documentation/no-changelog` so it's easier to find on the Github side